membership-common/conf/touchpoint.CODE.conf

```diff
@@ -82,7 +82,7 @@ touchpoint.backend.environments {
                     digipack="2c92c0f84b786da2014b91d3629b4298"
                     supporterPlus="8ad09fc281de1ce70181de3b23b2363d"
                     tierThree="8ad097b48ff26452019001c67ad32035"
-                    guardianLight="8ad09658929f4cc80192afc6ec08389e"
```

Comment:
@david-pepper new Product ID for CODE: 8ad0869c9444afc7019446c5eaf33503

Reply:
Done